Formula/code-server.rb

```diff
@@ -0,0 +1,25 @@
+require "language/node"
+
+class CodeServer < Formula
+  desc "Access VS Code through the browser"
+  homepage "https://github.com/cdr/code-server"
+  url "https://registry.npmjs.org/code-server/-/code-server-3.3.0-rc.7.tgz"
+  sha256 "22b7df81f2a5d372bac5bf5251407ca18493675ee3d50113b4d7817b711503b2"
+
+  depends_on "python" => :build
+  depends_on "yarn" => :build
+  depends_on "node"
+
+  def install
+    system "yarn", "--production"
```

Comment:
Why not use the npm functions in homebrew?

Reply:
We use yarn, don’t have a package lock json.

Reply:
Fine with me 👍